core/src/main/java/org/apache/accumulo/core/clientImpl/ThriftTransportPool.java

```diff
@@ -431,6 +433,10 @@ void shutdown() {
       }
     }
 
+    boolean awaitShutdown(long timeoutMillis) throws InterruptedException {
```

Comment:
If `shutdownLatch` is reachable from where this is called, then we could remove this method.

Reply:
yes, everything is reachable, as it's all in one compilation unit. 
i thought that with this method this is more readable.

Reply:
Fair enough.